image_rotation.py

```diff
@@ -0,0 +1,172 @@
+'''
+Author: Kilando Chambers
+
+This module contains the image rotation class that can be used to rotate the
+an image containing a streak such that the streak is horizontal, or parllel with
+the x-axis.  Use the line_detection_testing.ipynb file to import this module and 
+apply it on multiple images.
+
+Process (in development currently):
+    1. Declare init variables (see description of parameters within the class)
+    2. Choose image
+    3. Process the image:
+        a. Apply z_score_trim once to reduce the outlier pixel values
+        b. Apply z_score_trim on this processed image again
+        c. Standardize and normalize the image using cv2
+        d. Perform Canny edge detection
+        e. Trim the edges of the image (if chosen)
+    4. Perform Hough Transformation on the Canny edge image to get streak coordinates
+    5. Convert streak polar coordinates into cartesian coordinates
+    6. Find the mean coordinates at the edges of the image
+    7. Determine the angle of rotation for the image based on the mean coordinates
+    8. Rotate the image at the determined angle of rotation
+
+Next steps from here:
+    Plot a histogram of the brightness of the image
+'''
+
+#Importing necessary libraries
```

Comment:
We don't usually need comments for the most obvious things, they are only useful when they clarify some complicated piece of math or a foor loop or something unexpected.